src/dotty/tools/dotc/core/tasty/TastyFormat.scala

```diff
@@ -126,17 +126,19 @@ Standard-Section: "ASTs" TopLevelStat*
                   TYPEREFsymbol         sym_ASTRef qual_Type
                   TYPEREFpkg            fullyQualified_NameRef
                   TYPEREF               possiblySigned_NameRef qual_Type
+                  RECtype               parent_Type
                   SUPERtype      Length this_Type underlying_Type
                   REFINEDtype    Length underlying_Type refinement_NameRef info_Type
                   APPLIEDtype    Length tycon_Type arg_Type*
-                  TYPEBOUNDS     Length low_Type high_Type
+                  TYPEBOUNDS     Length low_Type high_Type bindingKind_Nat?
                   TYPEALIAS      Length alias_Type (COVARIANT | CONTRAVARIANT)?
                   ANNOTATED      Length underlying_Type fullAnnotation_Term
                   ANDtype        Length left_Type right_Type
                   ORtype         Length left_Type right_Type
                   BIND           Length boundName_NameRef bounds_Type
                                         // for type-variables defined in a type pattern
                   BYNAMEtype            underlying_Type
+                  LAMBDAtype     Length result_Type NamesTypes      // variance encoded in front of name: +/-/=
```

Comment:
It's pickled as an APPLIEDtype

On Tue, Jun 28, 2016 at 10:49 PM, Guillaume Martres <
notifications@github.com> wrote:

> In src/dotty/tools/dotc/core/tasty/TastyFormat.scala
> <https://github.com/lampepfl/dotty/pull/1337#discussion_r68840351>:
>
> >                    TYPEALIAS      Length alias_Type (COVARIANT | CONTRAVARIANT)?
> >                    ANNOTATED      Length underlying_Type fullAnnotation_Term
> >                    ANDtype        Length left_Type right_Type
> >                    ORtype         Length left_Type right_Type
> >                    BIND           Length boundName_NameRef bounds_Type
> >                                          // for type-variables defined in a type pattern
> >                    BYNAMEtype            underlying_Type
> > +                  LAMBDAtype     Length result_Type NamesTypes      // variance encoded in front of name: +/-/=
>
> Do we never need to pickle HKApply ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lampepfl/dotty/pull/1337/files/4377aaa94628ed8afffdc4801e1c83996832e21c#r68840351>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAwlVj1eixuFsgVJMW6OCsdS2nIW1JM6ks5qQYjmgaJpZM4JAdeo>
> .
>



-- 
Prof. Martin Odersky
LAMP/IC, EPFL